### PR TITLE
feat(options) Apply the appropriate update channel when an option is set

### PR DESCRIPTION
**Pull request by fpacifici**
With the options automator works we have to detect drift between options on the DB and options in github. 
In order to do this we record the author of the last update on the DB when that happens.
The default value is `UNKNOWN`.  
This PR sets that parameter correctly across the code base. 

The expected change is that, when an option is set, the new `last_update_channel` column in the DB options table will be populated.

### CODE REVIEW

File: src/sentry/runner/commands/config.py [hunk 1]
```diff
@@ -44,9 +44,13 @@ def get(option, silent):
     # TODO(mattrobenolt): Add help to option keys
     # if key.help:
```
Comment (EricHasegawa) [inline src/sentry/runner/commands/config.py:45]: Can we take out commented out code?

**Review by EricHasegawa — COMMENTED** [review]
Minor comment - also seems like tests need to be added, is there an exception here?

File: src/sentry/runner/commands/config.py (hunk 1 above)
Comment (fpacifici) [reply src/sentry/runner/commands/config.py]: yeah, will remove. Don't kjnow why it is there

**Comment by fpacifici** [comment]
Added a few tests

**Review by EricHasegawa — APPROVED** [review]

**Pull request merged**